test/sourcemaps/samples/preprocessed-script/test.js

```diff
@@ -0,0 +1,32 @@
+export function test({ assert, input, js }) {
+	const expectedBar = input.locate('baritone:');
+	const expectedBaz = input.locate('baz:');
+
+	let start = js.locate('bar:');
+
+	const actualbar = js.mapConsumer.originalPositionFor({
+		line: start.line + 1,
+		column: start.column
+	});
+
+	assert.deepEqual(actualbar, {
+		source: 'input.svelte',
+		name: null,
+		line: expectedBar.line + 1,
+		column: expectedBar.column
+	}, "couldn't find bar: in source");
```

Comment:
I'm surprised this passes linting. Strings should generally be contained in single quotes

Reply:
the eslint rules allow double quotes if the content of the string contains a single quote
```js
quotes: ['error', 'single', { avoidEscape: true }],
```